### PR TITLE
Added the community members page to the site

### LINKED ISSUE #579: Community member profile page
**Current Behavior**
Individual pages are generated for each person in the [_members collection](https://github.com/layer5io/layer5/tree/master/collections/_members) at https://layer5.io/community/members/<member-name>. Each member page is blank, though, because there is no HTML snippet that has been created to present a member's profile (e.g.,`{% include member.html %}` ).

**Desired Behavior**
Create a `member-html` and a `members-list.html` in order to present the "_members" collection and display a member's profile.

**Related Issue: #578**

---
**Resources**

This issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

**Description**

This PR fixes #579 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: _data/navigation.yaml [hunk 1]
```diff
@@ -36,6 +36,9 @@
     - name: MeshMates
       link: /community/meshmates
       new_window: false
+    - name: Members
```
Comment (leecalcote) [inline _data/navigation.yaml:39]: I'd like to leave this link outside of the main menu. Arguably, the MeshMates link could be removed as well. We can wait to do that for the layer5ng site, though.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thank you!

A couple items:

- social icons are drifting away from the profile card
- the profile pictures are not retaining their aspect ratio

Screen Shot 2020-09-03 at 4 22 16 PM

File: _data/navigation.yaml (hunk 1 above)
Comment (Nikhil-Ladha) [reply _data/navigation.yaml:39]: Okay,  I will see if it can be inserted in the community page somewhere, or not.

**Comment by leecalcote** [comment]
Getting closer. 

<a href="https://ibb.co/rcc7dbp">82942731-07-DB-48-A0-B757-19493-E69-B9-A0</a>

**Comment by Nikhil-Ladha** [comment]
> Getting closer. Here’s a screenshot from an iPhone:
> 
> [82942731-07-DB-48-A0-B757-19493-E69-B9-A0](https://ibb.co/rcc7dbp)

Didn't understand what's the issue?

**Comment by leecalcote** [comment]
The aspect ratio on the pictures is not retained. The pictures are skewed (the height and width dimensions are being adjusted independent of one another, but need to be in lockstep). The css on the meshery.io/community page uses overflow to deal with this, I believe.

**Comment by Nikhil-Ladha** [comment]
> The aspect ratio on the pictures is not retained. The pictures are skewed (the height and width dimensions are being adjusted independent of one another, but need to be in lockstep). The css on the meshery.io/community page uses overflow to deal with this, I believe.

Ohh....Nah, it's not the overflow, rather the `object-fit` property is doing the trick. Updating it...

**Comment by Nikhil-Ladha** [comment]
@leecalcote check now

**Comment by Nikhil-Ladha** [comment]
Which one looks better?
Screenshot from 2020-09-07 18-00-02
Screenshot from 2020-09-07 18-00-07

For the active/inactive part..

**Comment by leecalcote** [comment]
The first one. :thu

**Review by leecalcote — DISMISSED** [review]
@Nikhil-Ladha I know you're still working on the inactive vs active styling.

This is a great update, though, and I can't wait to have this first bit on the site. Please do follow up with an update for inactive vs active.

Nice work here. Looks great. 😄

**Review by leecalcote — APPROVED** [review]

**Pull request merged**